README.md

```diff
@@ -215,6 +215,60 @@ The possible values for the annotation can be
 * `"my-instrumentation"` - name of `Instrumentation` CR instance.
 * `"false"` - do not inject
 
+#### Use Customized Image for auto-instrumentation injection
+
+Although the default auto-instrumentation image can handle some mostly used libraries, it may still 
+miss the ones you need. In this case, you can tell the operator to use your own customized image.
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: my-instrumentation
+spec:
+  exporter:
+    endpoint: http://otel-collector:4317
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+  sampler:
+    type: parentbased_traceidratio
+    argument: "0.25"
+  java:
+    image: your-customized-auto-instrumentation-image:java
+  nodejs:
+    image: your-customized-auto-instrumentation-image:nodejs
+  python:
+    image: your-customized-auto-instrumentation-image:python
+```
+
+To build your customized image, you can start with editing the default dockerfiles. They are stored in the `autoinstrumentation` 
```

Comment:
> you can start with editing the default dockerfiles.

Replace "with editing" with "by reusing"? Editing could make people assume that they have to commit them back or maintain a forked repository.

Reply:
updated